d/doctrine__reflection/doctrine-reflection_1.1.0_1_rhel_8.4.sh

```diff
@@ -0,0 +1,50 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: Doctrine-Reflection 
+# Version	: 1, 1.1.0
```

Comment:
As per line 18, build script is created for version 1. Please update accordingly

Reply:
The script is tested and working fine on rhel 8.4. Also parameterized the version in line 18.